_data/accessibility-tests/select.yml

```diff
@@ -0,0 +1,113 @@
+  - summary: Select component remains functional when screen is magnified.
+    summary_additional: |
+      When you view the select and zoom to 400%,
+      the select list remains both visible and functional.
+    test_status: pass
+    test_type: zoom
+    version_tested: 3.8.0
+    wcag_criterion: 1.4.10
+# Keyboard navigation tests
+  - summary: All features of the select element are keyboard operable.
+    summary_additional: |
+      When using a keyboard,
+      you are able to tab into the select element, arrow down to select an option, and hit enter to select.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.0
+    wcag_criterion: 2.1.1
+  - summary: The focus indicator appears around every element in the select component.
+    summary_additional: |
+      When navigating to select,
+      you can see the focus indicator appear around the select component and on options within the select dropdown.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.0
+    wcag_criterion: 2.4.7
+  - summary: Navigating to select doesn't trigger a change of context.
+    summary_additional: |
+      When you type into the select,
+      the context of the select component doesn't change.
+      For example, landing on select doesn't cause a window to open or focus to jump to another field.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.0
+    wcag_criterion: 3.2.1
+# Screen reader tests
+  - summary: Screen reader announces select dropdown items in a list.
+    summary_additional: |
+      When you navigate with arrow keys,
+      `enter` and `spacebar` key while using a screen reader, you hear the the drop down label and items in the list.
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.8.0
+    wcag_criterion: 1.3.1
+  - summary: Select options are read in the same order that they appear on the page.
+    summary_additional: |
+      When you use a screen reader and navigate through the dropdown list,
+      you hear each text option in the same order in which they appear on the page.
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.8.0
+    wcag_criterion: 1.3.2
+  - summary: Screen reader provides information about the purpose of the select.
+    summary_additional: |
+      When you are using a screen reader,
+      you will know the common purpose of the select option (e.g., "Select your state").
+    test_status: conditional
+    test_type: screen_reader
+    version_tested: 3.8.0
+    wcag_criterion: 1.3.5
+  - summary: Screen reader announces user is in a select element.
+    summary_additional: |
+      When you use a screen reader and access the select component,
+      you will hear that you have entered a "combo box", "select",  or "menu pop-up" element,
+      and that you can select an option.
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.8.0
+    wcag_criterion: 2.4.6
```

Comment:
We've also discussed removing this test since it requires page headings and labels to be descriptive. This could be considered more page centered and removed from this test, or potentially moved to the general/content test section.

`1.3.1` handles screen reader callouts for component labels

Reply:
@amycole501 @alex-hull Do you have a preference on this? Is it ok to move this to the general section? If so, we'll need to re-write a bit

Reply:
Yes let's move it to general and outside of the screen reader tests.

Reply:
I am stumped on how to re-write 2.4.6 to be a general test tbh. It seems like it could be re-written to something like "Select component has descriptive labels", but I feel like it is covered by the update I just did to
[3.3.2](https://github.com/uswds/uswds-site/pull/2702#discussion_r1640239493). Is it possible to just remove 2.4.6 in favor of 3.3.2, or is there a key distinction I'm missing?

Reply:
Based on internal conversation, I updated the summary to match combo box's summary for the same criterion in [ba21517](https://github.com/uswds/uswds-site/pull/2702/commits/ba21517de6f5f797d3b0c807ff7562aecdea3e89): 

> **Select component offers visual and auditory instructions.** When you view or interact with the select component, you hear details about the purpose of the input with a screen reader and see them on the page.The instructions are specific to the component's intent (e.g., "Select a state", "Select a fruit").